SimPEG/utils/model_utils.py

```diff
@@ -260,3 +260,73 @@ def ismember(a, b):
     actv[inds] = True
 
     return actv
+
+
+    def depth_weighting(mesh, topo, indActive, v, z0):
+    """
+    Create depth weighting
+
+    Parameters
+    ----------
+    mesh : Mesh object
+        discretize model space.
+    topo : numpy.ndarray
+        [X,Y,Z] topographic data.
+    indActive : array of bool
+        index vector for the active cells on the mesh below the topography
+    v : float
+        exponent constant parameter for depth weighting.
+    z0 : float
+        adjustable constant parameter.
+
+    Returns
+    -------
+    wz : numpy.ndarray
+        depth weighting captures decay of the potential field data.
+
+
+    ..math::
+
+        w(z) = (z + z0) ** (-0.5*v)
+
+        where z is depth of model cells along the z direction;
+        v and z0 are two adjustable parameters.
+
+
+    """
+
+    # Find the highest topo position
+    temp = np.abs(topo[:,2].max() - mesh.vectorNz)
+    index = temp.argmin()
+
+    # Move the zero elevation to the highest topo position
+    factor = mesh.vectorNz[index]
+
+    wz = np.power(
+        -mesh.vectorCCz + factor + z0,
+        -v
+        )
+
+    wz = wz ** 0.5
+
+    # Normalize
+    wz /= np.nanmax(wz)
```

Comment:
normalization should happen after applying the active indices

Reply:
Thanks